Documentation/tutorial/intro_toc.md

```diff
@@ -73,25 +74,43 @@ Relative path in detail
 ---------------
 If a *TOC Item* is linking to some relative path, there are three cases:
 
-1. Linking to another *TOC File*, for example, `href: examples/toc.md`.
+1. **Deprecated** ~~Linking to another *TOC File*, for example, `href: examples/toc.md`.~~
 2. Linking to a folder, which means, the value of the link ends with `/` explicitly, for example, `href: examples/`
```

Comment:
as discussed, remain the current usage

---
In reply to: [72391784](https://github.com/dotnet/docfx/pull/537#discussion_r72391784) [](ancestors = 72391784)